augur/utils.py

```diff
@@ -551,3 +552,13 @@ def read_strains(*files, comment_char="#"):
                     strains.add(strain_name)
 
     return strains
+
+@enum.unique
+class DataErrorMethod(enum.Enum):
+    """
+    Enum representaion of string values that represent how a data error should
+    be handled.
+    """
+    ERROR   = 'error'
+    WARN    = 'warn'
+    SILENT  = 'silent'
```

Comment:
I'm not a fan of `utils.py` being a dumping ground for unrelated things, however I understand that sometimes it's difficult to find better placement.

For `DataErrorMethod` specifically, how about moving it to `augur/io/print.py`? Since it is describing different methods of printing.

Reply:
Would it make sense to move to this to `augur/errors.py`?

Reply:
In that it's about errors, yeah, though it's not an error object / exception class. I'd vote `augur/errors.py` or `augur/types.py` (this enum class is really a data type).

Reply:
`augur/types.py` makes sense!

Reply:
Added to `augur/types.py` in bace92ba7a11e5a2d6466fa784f93b878051677d